channels/mx.m3u

```diff
@@ -5,8 +5,6 @@ http://live.gvstream.net:1935/e-tv-video/play.stream/playlist.m3u8
 http://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master01.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/9Xn7ZEZ.jpg" group-title="",Alcarria TV
 http://217.182.77.27/live/alcarriatv-livestream.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/rbDb5mb.png" group-title="",Aprende TV
-https://srvstream01.televisioneducativa.gob.mx/hls/aprendetv.m3u8
```

Comment:
Link works (Tested in VLC)

Reply:
Still not working. VLC output:
<img width="738" alt="Screenshot 2019-10-12 at 20 14 22" src="https://user-images.githubusercontent.com/7253922/66705260-fc5fee00-ed2c-11e9-829a-24e99d18f937.png">

Reply:
This stream works for me in VLC 
![Screenshot_2019-10-12 Test Live Streaming Streamtest in(1)](https://user-images.githubusercontent.com/25525836/66706615-3503c580-ed35-11e9-9abc-006c2748f7db.png)

Reply:
All the links starting with "https://srvstream01.televisioneducativa.gob.mx/hls"
are from this website 
https://www.televisioneducativa.gob.mx/#multimedia
try if you can play them there.

Reply:
I can’t access this site at all
<img width="684" alt="Screenshot 2019-10-13 at 04 38 35" src="https://user-images.githubusercontent.com/7253922/66709761-67341800-ed73-11e9-8551-5fe499efa6b7.png">